src/main/java/us/icarenow/web/config/MvcConfig.java

```diff
@@ -0,0 +1,16 @@
+package us.icarenow.web.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class MvcConfig implements WebMvcConfigurer {
+
```

Comment:
this is not required, spring mvc takes index.html as default root page.
Please remove this class